easybuild/toolchains/linalg/blis.py

```diff
@@ -42,3 +44,15 @@ class Blis(LinAlg):
     BLAS_MODULE_NAME = ['BLIS']
     BLAS_LIB = ['blis']
     BLAS_FAMILY = TC_CONSTANT_BLIS
+
+    def _set_blas_variables(self):
+        """AMD's fork with version number > 2.1 names the MT library blis-mt, while vanilla BLIS doesn't."""
+
+        # This assumes that AMD's BLIS has ver > 2.1 and vanilla BLIS < 2.1
+
+        found_version = self.get_software_version(self.BLAS_MODULE_NAME)[0]
+        ver = LooseVersion(found_version)
+        if ver > LooseVersion('2.1'):
```

Comment:
no need to create a variable if it's only used once, so reduce to
if LooseVersion(found_version) > LooseVersion('2.1'):

Reply:
Thanks for the comment! I changed it.